content/extraProblems.json

```diff
@@ -66,6 +66,18 @@
         "usacoId": "812"
       }
     },
+    {
+      "uniqueId": "usaco-856",
```

Comment:
oops idk why i didn't realize this but this problem is already [here](https://github.com/cpinitiative/usaco-guide/blob/master/content/2_Bronze/Simulation.problems.json#L83) so it doesn't have to be added to extraProblems.json; just edit that file and change `kind` to `internal` and remove `usacoId`

Reply:
> oops idk why i didn't realize this but this problem is already [here](https://github.com/cpinitiative/usaco-guide/blob/master/content/2_Bronze/Simulation.problems.json#L83) so it doesn't have to be added to extraProblems.json; just edit that file and change `kind` to `internal` and remove `usacoId`

@AAkashyap1 please change this immediately.

Reply:
Was this successfully changes @maggie-j-liu

Reply:
yep, it looks fine now